src/mock-doc/node.ts

```diff
@@ -138,6 +138,7 @@ export class MockNode {
   }
 
   contains(otherNode: MockNode) {
+    if (otherNode === this) return true;
```

Comment:
Super minor nitpick - could you do me a favor and wrap the `return` statement in curly braces for me please? 
```suggestion
    if (otherNode === this) {
      return true;
    }
```